index.html

```diff
@@ -5,104 +5,128 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="mobile.css">
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,700&amp;subset=latin-ext" rel="stylesheet">
-    <title>strona</title>
+    <link rel="stylesheet" href="nav.css">
+    <link rel="stylesheet" href="header.css">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="section.css">
+    <link rel="stylesheet" href="footer.css">
+    <link href="https://fonts.googleapis.com/css?family=Oxygen:400,700&amp;subset=latin-ext" rel="stylesheet">
+    <title>dokument</title>
 </head>
 
 
 <body>
-    <nav class="logo">
-                <a href="" class="logo1">market</a>           
-                <a href="" class="logo2">buy now</a>
-                <a href="" class="logo3">Remove frame</a>
+    
+    <nav class="navbar">
+        <div class="navbar-branding">
+            <a href="" class="logo2">envato</a>
+        </div>
+        <div class="navbar-list">
+            <div><a href="" class="logo3">buy now</a></div>
```

Comment:
Pytanie czy musimy opakowywać `a` w `div`? Dlaczego chcemy to zrobić? Czy wiemy dlaczego po opakowaniu w `div` coś nagle zaczyna działać tak jakbyśmy chcieli? ;) Nie jest to jakiś błąd, ale załóżmy, że chcemy pozbyć się `div`ów obejmujących elementy `logo3` oraz `logo4` - co musimy zrobić, żeby inaczej wyśrodkować w pionie oba elementy? ;) (w niektórych przypadkach opakowanie coś w `div` ma sens, ale w tym przypadku jest to raczej zbędne)